### PR TITLE
Ajout Yannick DALBIN

### PR DESCRIPTION
**Pull request by YannickDa**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -15440,6 +15440,92 @@ var data = {
...
+        {
+          "name": "Web",
+          "url": "https://startupmarseille.com"
+        },
+        {
+          "name": "Web",
+          "url": "https://youtube.com/yannickdalbin"
+        },
+        {
+          "name": "Web",
```
Comment (dadoonet) [inline baggers.js:15467]: I don't think it's needed as we have later the twitter handle

File: baggers.js [hunk 2]
```diff
@@ -15440,6 +15440,92 @@ var data = {
...
+          "url": "https://twitter.com/yannickda"
+        },
+        {
+          "name": "Web",
```
Comment (dadoonet) [inline baggers.js:15471]: Note to self: should we add Facebook?

**Comment by nrichand** [comment]
Welcome Yannick

File: baggers.js [hunk 3]
```diff
@@ -15533,6 +15619,12 @@ var data = {
       "lat": 45.764043,
       "lng": 4.835659
     },
+    "Marseille": {
```
Comment (dadoonet) [inline baggers.js:15622]: What? You are the first in Marseille? :)

**Comment by YannickDa** [comment]
Merci :)

File: baggers.js [hunk 4]
```diff
@@ -15440,6 +15440,92 @@ var data = {
...
+            "Javascript",
+            "Webpack"
+          ],
+          "lang": [
+            "fr"
+          ]
+        }
+      ],
+      "cities": [
+        "Marseille"
```
Comment (dadoonet) [inline baggers.js:15523]: I wonder if you would like to speak as well in Aix en Provence? May be add it here?

**Comment by dadoonet** [comment]
Left some comments. Feel free to adapt or not :)

File: baggers.js (hunk 4 above)
Comment (YannickDa) [reply baggers.js:15523]: It's is a little far away for me... In Marseille, we have lot of cars and not enough bus or something like that :)

File: baggers.js (hunk 3 above)
Comment (YannickDa) [reply baggers.js:15622]: haha :)

File: baggers.js (hunk 2 above)
Comment (nrichand) [reply baggers.js:15471]: it's so 2016... :rofl:

Comment (YannickDa) [reply baggers.js:15471]: You wrong :)

Facebook make lot of work on Pages ! It's become a powerfull tools !

It's so 2017 :)

**Pull request merged**